src/EnergyPlus/UnitarySystem.cc

```diff
@@ -2493,6 +2745,15 @@ namespace UnitarySystems {
         }
         if (this->m_HeatingCoilType_Num == DataHVACGlobals::Coil_HeatingWater) {
 
+            // pass air flow rate to zone water coil
+            if (state.dataSize->CurZoneEqNum > 0) {
+                WaterCoils::SetCoilDesFlow(state,
+                                           DataHVACGlobals::cAllCoilTypes(this->m_HeatingCoilType_Num),
+                                           this->m_HeatingCoilName,
+                                           this->m_MaxHeatAirVolFlow,
+                                           state.dataUnitarySystems->initUnitarySystemsErrorsFound);
+            }
+
```

Comment:
Regarding the changes to results in UnitarySytsem unit test UnitarySystemModel_ASHRAEModel_WaterCoils. This new code now passes the air flow rate to the water heating coil (water heating coil does not have a design air flow rate). If this is not done then in CalcAdjustedCoilUA the UACoil gets used, which does not get adjusted for air and water flow and temp difference from design (and we are never warned about that?).